### PR TITLE
update Rhodes app copy

### LINKED ISSUE #2598: Update the copy in the membership update success snackbar component
The idea captured in the spreadsheet
> Not related to the test task, but I think it's worth mentioning. When I changed my membership avatar, I got a snackbar notification with lorem ipsum as a copy. ~~Perhaps it's worth to review copy of the whole platform, not just NFT's itself.~~ (Done)

CleanShot 2022-04-29 at 16 42 44

---
 
If I may suggest a copy right away, keeping in mind the other recent NFT-related copy changes, I'd suggest to ditch the body copy altogether and updating the header to say "Profile updated successfully". CC @dmtrjsg @kdembler @TCzechowski

### PR DESCRIPTION
**Pull request by kdembler**
Resolves #2598 

Almost all of the changes were agreed with the design team. Probably still worth taking a quick look to make sure I didn't mess anything up :D

### CODE REVIEW

File: packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx [hunk 1]
```diff
@@ -254,21 +254,21 @@ export const NftWidget: React.FC<NftWidgetProps> = ({
...
-                'You placed a bid in a previous auction that you can now withdraw to claim back your money and to be able to participate in this auction.',
+                'You placed a bid in a previous auction that you can now withdraw to be able to participate in this auction.',
             }
           }
 
           if (nftStatus.englishTimerState === 'expired' && isOwner && !hasBids) {
             return {
               title: 'Auction ended',
-              description: 'This auction has ended, but no one made an offer. You can now remove this NFT from sale.',
+              description: 'This auction has ended and no one placed a bid. You can now remove this NFT from sale.',
```
Comment (rafalpawlow) [inline packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx:264]:
```suggestion
              description: 'This auction has ended, and no one placed a bid. You can now remove this NFT from sale.',
```

File: packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx [hunk 2]
```diff
@@ -254,21 +254,21 @@ export const NftWidget: React.FC<NftWidgetProps> = ({
...
             }
           }
           if (nftStatus.englishTimerState === 'expired' && !bidFromPreviousAuction && !hasBids && !isOwner) {
             return {
               title: 'Auction ended',
               description:
-                "This auction has ended, but no one made an offer. We're waiting for the NFT owner to cancel the auction.",
+                "This auction has ended and no one placed a bid. We're waiting for the NFT owner to remove this NFT from sale.",
```
Comment (rafalpawlow) [inline packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx:271]:
```suggestion
                "This auction has ended, and no one placed a bid. We're waiting for the NFT owner to remove this NFT from sale.",
```

File: packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx [hunk 3]
```diff
@@ -54,61 +43,62 @@ export const AcceptTerms: React.FC<AcceptTermsProps> = ({
...
-      <Text variant="h400">Listing settings</Text>
+      <Text variant="h400">Listing terms</Text>
       <Row>
         <Title>
           <TitleText>Listing type</TitleText>
           <StyledInformation
-            text="Offers can be received. You can accept any bid at any time unless auction duration is set (then the highest offer wins)"
+            text={
+              isAuction
+                ? 'Put it on a timed or open auction. See the bids coming.'
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx:55]:
```suggestion
                ? 'Put it on a timed, or open auction. See the bids coming.'
```

File: packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx [hunk 4]
```diff
@@ -54,61 +43,62 @@ export const AcceptTerms: React.FC<AcceptTermsProps> = ({
...
       {formData?.buyNowPrice && (
         <Row>
           <Title>
-            <TitleText>Fixed price</TitleText>
+            <TitleText>{isAuction ? 'Buy now price' : 'Fixed price'}</TitleText>
             <StyledInformation
-              text="Sell your NFT for a predefined price. When this price is reached it automatically ends auction"
+              text={
+                isAuction
+                  ? 'Bids matching this value will automatically end your auction'
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx:85]:
```suggestion
                  ? 'Bids matching this value will automatically end your auction.'
```

File: packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx [hunk 5]
```diff
@@ -54,61 +43,62 @@ export const AcceptTerms: React.FC<AcceptTermsProps> = ({
...
+                  : 'Price for your NFT as shown to your buyers'
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx:86]:
```suggestion
                  : 'Price for your NFT as shown to your buyers.'
```

File: packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx [hunk 6]
```diff
@@ -54,61 +43,62 @@ export const AcceptTerms: React.FC<AcceptTermsProps> = ({
...
         </Row>
       )}
       {selectedType !== 'Fixed price' && (
         <Row>
           <Title>
-            <TitleText>Starting date</TitleText>
+            <TitleText>Start date</TitleText>
             <StyledInformation
-              text="It’s the time when your auction will become active and buyer will be able to make an offer"
+              text="The moment in time when your auction becomes active and bids will be accepted"
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx:101]:
```suggestion
              text="The moment in time when your auction becomes active, and bids will be accepted."
```

File: packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/ListingType/ListingType.tsx [hunk 7]
```diff
@@ -26,15 +26,15 @@ export const ListingType: React.FC<ListingTypeProps> = ({ selectedType, onSelect
           value="Auction"
           label="Auction"
           icon={<SvgActionAuction />}
-          helperText="Sell it on auction or for fixed price. Optionally choose time, whitelist, royalties."
+          helperText="Put it on a timed or open auction. See the bids coming."
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/ListingType/ListingType.tsx:29]:
```suggestion
          helperText="Put it on a timed, or open auction. See the bids coming."
```

File: packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/SetUp/SetUp.tsx [hunk 8]
```diff
@@ -168,14 +163,14 @@ export const SetUp: React.FC<SetUpProps> = ({
                   <OptionCardRadio
                     value="open"
                     label="Open auction"
-                    helperText="Cancel anytime & pick the winning bid"
+                    helperText="Pick the winning bid or cancel anytime"
```
Comment (rafalpawlow) [inline packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/SetUp/SetUp.tsx:166]:
```suggestion
                    helperText="Pick the winning bid, or cancel anytime"
```

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
I fell we have a little mess here with the periods at the end of the sentences, in some place we use and in some not. TBH I'm not sure what approach we should take.

**Comment by kdembler** [comment]
This copy has been written by Dima, Adam and me, and we discussed both periods and commas. I would personally skip those commas before or/and. Also, our approach to periods was that we place them only if the message was sentence-like, we tried to be as consistent as possible. I'm gonna disagree with your suggestions 😅

**Review by rafalpawlow — APPROVED** [review]
@kdembler Tbh I was sure that my comma suggestions followed the rules of grammar. But this recommendation I trust 😅
> This copy has been written by Dima, Adam and me

**Pull request merged**